portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+/** Add comment */
+public final class Comment {
+  private final String email;
+  private final String comment;
+
+  public Comment(String email, String comment) {
+    this.email = email;
+    this.comment = comment;
+  }
+}
```

Comment:
What do you think about creating a static method called fromEntity(Entity entity) that allows you to create a comment from an Entity object?

See below for the cool things you can do with this method.

Reply:
I am not sure what you mean or how to do it. 

Where below for cool things I can do with this method?

Reply:
This was supposed to be above!

If you create a static fromEntity method in Comment, you can use fancy Java (streams and method reference) to build an ArrayList<Comment> in one line:

results.asIterable().stream().map(Comment::fromEntity).collect(Collectors.toCollection(ArrayList::new));

No need to implement, just wanted to show you something new :)